test/functional/examples.js

```diff
@@ -0,0 +1,87 @@
+/* View */
```

Comment:
What is the purpose of this file?

Reply:
I removed this file from this PR. It was a proposition of reorganisation of the functionnal test files.